Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM centos:8.2.2004 as katapackages
+ARG ARCH=amd64
+FROM ${ARCH}/centos:8.2.2004 as katapackages
```

Comment:
Hi @fidencio the ARCH argument is not mandatory and can be removed. Centos image in dockerhub is multi-arch. So using the image without any arch prefix will work for both Intel and Power. 
For example on ppc64le
```
docker run -it centos:8.2.2004 uname -m
ppc64le
```
on amd64 
```
docker run -it centos:8.2.2004 uname -m
x86_64
```
A good description is available in the following [link](https://developer.ibm.com/tutorials/createmulti-architecture-docker-images/)

I have also tested building the container image on standalone ppc64le successfully using your changes.

Reply:
Cool, I've dropped the ARCH and re-worked this series to use buildx.

Thanks for pointing this out!